app/views/pet_schedules/new.html.erb

```diff
@@ -26,10 +26,11 @@
 
 <div class="sf-outfit-schedul">
   <div class="card-product">
```

Comment:
not super important, but maybe you can change the name of the class the reflect our project ;) it can stay like this too, your call

Reply:
Haha got lazy last night, I will change name tonight lol

Reply:
ok, no problem!